src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -57,44 +56,49 @@ static ColliderSphereInit sSphereInit = {
     { 0, { { 0, 0, 0 }, 200 }, 100 },
 };
 
+typedef enum {
+    /* 0x0 */ POFUSEN_DAMAGEEFFECT_NOEFFECT,
```

Comment:
I think we've usually been going with `NONE` for these instead of `NOEFFECT`.

Reply:
oh I ripped that from one of my old actors, will change.

Reply:
Ah, yeah, `EnPoSisters` happens to be the only actor in the whole codebase with `NOEFFECT`.